modules/spring-config/src/main/java/org/mule/runtime/config/spring/ModuleDelegatingEntityResolver.java

```diff
@@ -56,7 +63,8 @@ public InputSource resolveEntity(String publicId, String systemId) throws SAXExc
 
   private InputSource generateModuleXsd(String publicId, String systemId) {
     InputSource inputSource = null;
-    if (extensionManager.isPresent()) {
+    if (extensionManager.isPresent()
```

Comment:
ExtensionManager could be absent, but the schemaResourceFactory must not.
I'll add the same TODO to the `if` condition so it gets removed once MULE-11024 is done